distribution/docs/src/main/resources/content/_metadataAttributes/NITF_TRE_STDIDC-table.adoc

```diff
@@ -18,7 +18,7 @@
 |Tre.STDIDC.COUNTRY
 |2char FIPS -> 3char ISO3
 
-If conversion fails other system plugins could populate the country code based on location.
+If conversion fails, then null. NOTE: Other system plugins could populate the country code based on location.
```

Comment:
![image](https://user-images.githubusercontent.com/8041246/43423525-89fee256-9401-11e8-8b4f-678bfc09b9e5.png)
I confirmed that this hyperlinks jumps to the right place.